instrumentation/cassandra/cassandra-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/cassandra/v3_0/CassandraSingletons.java

```diff
@@ -21,18 +23,24 @@ public final class CassandraSingletons {
   // could use RESPONSE "ResultSet" here, but using RESPONSE "ExecutionInfo" in cassandra-4.0
   // instrumentation (see comment over there for why), so also using here for consistency
   private static final Instrumenter<CassandraRequest, ExecutionInfo> INSTRUMENTER;
+  private static final AttributeKey<String> DB_COLLECTION_NAME = AttributeKey.stringKey("db.collection.name");
 
   static {
     CassandraSqlAttributesGetter attributesGetter = new CassandraSqlAttributesGetter();
 
+    AttributeKey<String> tableAttribute = DbIncubatingAttributes.DB_CASSANDRA_TABLE;
+    if (SemconvStability.emitStableDatabaseSemconv()) {
+      tableAttribute = DB_COLLECTION_NAME;
+    }
```

Comment:
```suggestion
    AttributeKey<String> tableAttribute = SemconvStabilityUtil.getAttributeKey(DbIncubatingAttributes.DB_CASSANDRA_TABLE);
```

Reply:
SemconvStabilityUtil is created for testing only.